### ISSUE TITLE
Request: Remove text-suggestions during video playback

### ISSUE THREAD

**Issue by Brog33** [lawrencehook/remove-youtube-suggestions#52]
These annoying text-suggestions still isn't removed (see screenshot).

![image](https://user-images.githubusercontent.com/1303537/162786822-efc24cc9-c90e-4314-a6e2-27e441b8da84.png)

**Comment by lawrencehook** [comment]
I don't see this text suggestion on my YouTube videos. Are you able to see what the css selector for it is?

**Comment by Brog33** [comment]
Ah ok. I have noticed them in a good few videos. The video i was screenshotting above was this one https://www.youtube.com/watch?v=XzPK-DpoP1M 

There seems to be at least two different variants. Managed to kill them off with ublock origin. 
Here are the rules from the picker.

www.youtube.com##.ytp-cards-button-icon-default > .ytp-cards-button-icon
www.youtube.com###movie_player > .ytp-chrome-top > .ytp-chrome-top-buttons > .ytp-cards-teaser
